tests/Feature/GraphQL/Mutations/CreateExportTest.php

```diff
@@ -37,4 +38,8 @@
     Event::assertDispatched(ExportInitialised::class);
 
     expect($response['status'])->toBe(ExportResponseStatus::SUCCESS);
+
+    $export = Export::latest('created_at')->first()->refresh();
+    expect($export->args['dateFrom'])->toBe('2021-01-01');
+    expect($export->args['dateTo'])->toBe('2021-01-02');
```

Comment:
maybe add a test that tries to break the date exports by fucking around with the timezone or messing with the input dates? If it makes sense?